modules/logging/aws-for-fluent-bit/variables.tf

```diff
@@ -0,0 +1,250 @@
+# For depends_on queue
+variable "module_depends_on" {
+  default = []
+}
+
+variable "namespace_name" {
+  description = "Name of the namespace where install charts"
+  default     = "logging"
+}
+
+variable "cluster_name" {
+  description = "Name of the kubernetes cluster"
+}
+
+variable "aws_region" {
+  description = "Fluent Bit target's AWS Region"
+}
+
+variable "cloudwatch_enabled" {
+  default     = false
+  description = "Whether to enable CloudWatch output"
+}
+
+variable "cloudwatch_match" {
+  default     = "'*'"
+  description = "The log filter"
+}
+
+variable "cloudwatch_log_group_name" {
+  default     = "/aws/eks/fluentbit-cloudwatch/logs"
+  description = "The name of the CloudWatch Log Group that you want log records sent to"
+}
+
+variable "cloudwatch_log_stream_name" {
+  default     = ""
+  description = "The name of the CloudWatch Log Stream that you want log records sent to."
+}
+
+variable "cloudwatch_log_stream_prefix" {
+  default     = "fluentbit-"
+  description = "Prefix for the Log Stream name"
+}
+
+variable "cloudwatch_log_key" {
+  default     = ""
+  description = "Log record key value of which will be sent to CloudWatch. By default whole message will be sent"
+}
+
+variable "cloudwatch_log_format" {
+  default     = ""
+  description = "Format of the log data"
+}
+
+variable "cloudwatch_role_arn" {
```

Comment:
Could you please create role and policy for all output options and auto setup this arn here?

Reply:
Done